lib/view.cc

```diff
@@ -172,6 +172,12 @@ View View::getMessages(const std::string &topic) {
 }
 
 View View::getMessages(const std::vector<std::string> &topics) {
+  std::chrono::nanoseconds start_time_ns {getStartTime().to_nsec()};
+  std::chrono::nanoseconds end_time_ns   {getEndTime().to_nsec()};
+  return getMessages(topics, start_time_ns, end_time_ns);
+}
+
+View View::getMessages(const std::vector<std::string> &topics, std::chrono::nanoseconds start_time, std::chrono::nanoseconds end_time) {
```

Comment:
I think it might make sense to operate on the data with `ros_time_t` and have that as the argument to the function, but then provide an easy to use constructor for `ros_time_t` from `std::chrono::nanoseconds`.